lib/axlsx/workbook/worksheet/cell.rb

```diff
@@ -172,7 +174,7 @@ def plain_string?
         !is_text_run? &&          # No inline styles
         !@value.nil? &&           # Not nil
         !@value.empty? &&         # Not empty
-        !@value.start_with?(?=)  # Not a formula
+        !@value.start_with?(*FORMULA_PREFIXES) # Not a formula
```

Comment:
I did not understand exactly what Shared Strings are, but:

1. Should this also check for an array formula?
2. Should this use `!is_formula?` itself

```suggestion
        !is_formula? &&           # Not a formula
        !is_array_formula?        # Not an array formula
```

Reply:
Done.